index.bs

```diff
@@ -1765,6 +1786,68 @@ object |transport|, run these steps.
 
 </div>
 
+# Interface `WebTransportRateControlFeedback` #  {#rate-control-feedback}
+
+A {{WebTransportRateControlFeedback}} contains information
+useful for the application to control the rate at which it sends data.
+
+<pre class="idl">
+interface WebTransportRateControlFeedback {
+  readonly attribute unsigned long long? sendRate;
+};
+</pre>
+
+## Internal slots ## {#rate-control-feedback-internal-slots}
+
+A {{WebTransportRateControlFeedback}} has the following internal slots.
+
+<table class="data" dfn-for="WebTransportRateControlFeedback" dfn-type="attribute">
+ <thead>
+  <tr>
+   <th>Internal Slot
+   <th>Description (<em>non-normative</em>)
+  </tr>
+ </thead>
+ <tbody>
+  <tr>
+   <td><dfn>`[[SendRate]]`</dfn>
+   <td class="non-normative">The rate at which the {{WebTransport}} dequeues and sends data,
+   which is the maximum rate the applcation should send at to avoid queuing.
+  </tr>
+ </tbody>
+</table>
+
+## Attributes ##  {#rate-control-feedback-attributes}
+
+: <dfn for="WebTransportRateControlFeedback" attribute>sendRate</dfn>
+:: The getter steps are to return [=this=]'s {{WebTransportRateControlFeedback/[[SendRate]]}}.
+   This is the rate at which queued data will be dequeued and sent by the user agent, in bits per second.
+   If the application wishes to avoid queuing, it should not send more than this rate.
```

Comment:
We try to avoid "should" except for normative statments, and we can't make normative statements on applications. How about:   
```suggestion
   If the application wishes to avoid queuing, it can send less than at this rate.
```
?